### PR TITLE
skaffold 1.5.0

### PR DESCRIPTION
**Pull request by akhilles**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/skaffold.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Skaffold < Formula
   desc "Easy and Repeatable Kubernetes Development"
   homepage "https://github.com/GoogleContainerTools/skaffold"
   url "https://github.com/GoogleContainerTools/skaffold.git",
-      :tag      => "v1.4.0",
-      :revision => "bb324609b4254ceb9f76d35eb03642a6a1807867"
+      :tag      => "1.5.0",
```
Comment (Bo98) [inline Formula/skaffold.rb]: Looks like the tag is actually `v1.5.0`.

```suggestion
      :tag      => "v1.5.0",
```

**Comment by Bo98** [comment]
Looks like their build script is broken in Go 1.14: https://github.com/GoogleContainerTools/skaffold/pull/3700

**Comment by akhilles** [comment]
There's an open PR for switching to Go 1.14: https://github.com/GoogleContainerTools/skaffold/pull/3775

**Comment by mjgallag** [comment]
@Bo98 does it make sense to change `go` to `go@1.13` to unblock this upgrade?
https://github.com/Homebrew/homebrew-core/blob/f91bd5897420099b8b338145f11e1721180f40ec/Formula/skaffold.rb#L16

**Comment by Bo98** [comment]
If a 1.14 patch is non-trivial and no release is imminent, sure that's fine. There should however be a comment clearly indicating when it can be raised to 1.14 (next update or whatever).

**Comment by chenrui333** [comment]
Yeah, I think we can ship and move back to go 1.14 after the next release.

**Review by Bo98 — APPROVED** [review]

**Pull request closed without merging**